libs/s25main/GamePlayer.h

```diff
@@ -90,7 +90,7 @@ class GamePlayer : public GamePlayerInfo
     GameWorld& GetGameWorld() { return world; }
     const GameWorld& GetGameWorld() const { return world; }
 
-    const MapPoint& GetHQPos() const { return hqPos; }
+    const MapPoint& GetHQPos() const noexcept { return hqPos; }
```

Comment:
It's a little weird that the gcc-9 job complained about this. I didn't change this code (and `GameWorldView::GetViewer`) and yet I have to "fix" this? I don't even necessarily think this is a valid suggestion (keep marking things noexcept just because they don't throw) and if it is, then there are hundreds of other functions like this.

Reply:
Moved troublesome calls to initializer list.